### PR TITLE
fix: mount stuck issue

### PR DESCRIPTION
**Pull request by umagnus**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

cleanup: use util wait function instead of wait package which can't use in exec command, referring to https://github.com/kubernetes-sigs/blob-csi-driver/pull/1278

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// TimeoutFunc returns output and error if an ExecFunc timeout
+type TimeoutFunc func() (err error)
+
+// WaitUntilTimeout waits for the exec function to complete or return timeout error
+func WaitUntilTimeout(timeout time.Duration, execFunc ExecFunc, timeoutFunc TimeoutFunc) error {
+	// Create a channel to receive the result of the azcopy exec function
+	done := make(chan bool)
+	var err error
+
+	// Start the azcopy exec function in a goroutine
```
Comment (andyzhangx) [inline pkg/util/util.go:35]: remove azcopy word

File: pkg/util/util.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+// ExecFunc returns a exec function's output and error
+type ExecFunc func() (err error)
+
+// TimeoutFunc returns output and error if an ExecFunc timeout
+type TimeoutFunc func() (err error)
+
+// WaitUntilTimeout waits for the exec function to complete or return timeout error
+func WaitUntilTimeout(timeout time.Duration, execFunc ExecFunc, timeoutFunc TimeoutFunc) error {
+	// Create a channel to receive the result of the azcopy exec function
```
Comment (andyzhangx) [inline pkg/util/util.go:31]: remove azcopy word

File: pkg/util/util.go (hunk 1 above)
Comment (umagnus) [reply pkg/util/util.go]: done

File: pkg/util/util.go (hunk 2 above)
Comment (umagnus) [reply pkg/util/util.go]: done

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

**Review by andyzhangx — APPROVED** [review]
/lgtm

**Pull request merged**